.github/workflows/terraform-docs.yaml

```diff
@@ -0,0 +1,24 @@
+name: terraform-docs
+run-name: generate terraform-docs ${{ github.event.pull_request.title }}
+on:
+  pull_request:
+    types: [ synchronize ]
```

Comment:
Why only `synchronize`?

Reply:
My thinking was that unlike fmt or validate whose requirements for passing may change upon reopening etc, terraform-docs only ever needs to be ran if there has been a commit to this branch.

Reply:
Is that what `synchronize` is?  I thought it was when you refresh from another branch (like update changes from `main`).

I suppose I should look it up.

Reply:
https://github.com/orgs/community/discussions/24567